numpy/core/fromnumeric.py

```diff
@@ -2406,6 +2409,20 @@ def any(a, axis=None, out=None, keepdims=np._NoValue, *, where=np._NoValue):
     >>> id(z), id(o) # identity of z and o              # doctest: +SKIP
     (191614240, 191614240)
 
+    >>> np.any([[1.0,3.5,-0.9],[4.5,7.9,0],[-5.6,11,3.7]],out=np.array(float))
+    array(1.0, dtype=object)
+    
+    >>> np.shape([[[1,2,4],[3,5,6]],[[3,4,5],[7,8,9]]])
+    (2,2,3)
+    >>> ex = np.any([[[1,2,4],[3,5,6]],[[3,4,5],[7,8,9]]],axis=2,keepdims=True)
+    >>> ex
+    array([[[ True],
+        [ True]],
+       [[ True],
+        [ True]]])
+    >>> ex.shape
+    (2,2,1)
+
```

Comment:
Do we care about the formatting of example in Docs, for example space after commas in parameters ?

```suggestion
    >>> np.any([[1.0,3.5,-0.9],[4.5,7.9,0],[-5.6,11,3.7]], out=np.array(float))
    array(1.0, dtype=object)
    
    >>> np.shape([[[1,2,4],[3,5,6]],[[3,4,5],[7,8,9]]])
    (2,2,3)
    >>> ex = np.any([[[1,2,4],[3,5,6]],[[3,4,5],[7,8,9]]], axis=2, keepdims=True)
    >>> ex
    array([[[ True],
        [ True]],
       [[ True],
        [ True]]])
    >>> ex.shape
    (2,2,1)

```

I'm not a core dev, so it's a real question but I personally find that with spaces (and having consistent code), it's easier to read.

Reply:
> Do we care about the formatting of example in Docs

yes